ibis/expr/visualize.py

```diff
@@ -80,7 +81,7 @@ def get_label(node):
 
 
 def to_graph(expr, node_attr=None, edge_attr=None, label_edges: bool = False):
-    graph = ibis.util.to_op_dag(expr.op())
+    graph = traverse.bfs(expr.op())
```

Comment:
I think I'd prefer the slightly-more-verbose `Graph.from_bfs(expr)` here, though I could also see a `Graph.from_expr(expr, algo=traverse.bfs)` 🤷🏻 

Not super opinionated here.

Reply:
Updated to `Graph.from_bfs(expr)`